samples/drivers/adc/boards/tlsr9518adk80d.overlay

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2022 Telink Semiconductor (Shanghai) Co., Ltd.
+ */
+
+#include <zephyr/dt-bindings/adc/adc.h>
```

Comment:
Wouldn't it be better to include it from `dts/riscv/telink/telink_b91.dtsi` instead?

Reply:
Done.